src/components/Header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+type Props = {
+  handleSubmit: (event: React.FormEvent) => void,
+  query: string,
+  setQuery: (title: string) => void,
+};
+
+export const Header:FC<Props> = ({ handleSubmit, query, setQuery }) => (
```

Comment:
The base filename 'Header' does not exactly match the name of its primary named export 'Header'. Consider changing the export to a default export to maintain consistency: `export default Header;`